src/js/tour.jsx

```diff
@@ -46,6 +46,8 @@ export class Tour extends Evented {
    * @param {boolean} options.useModalOverlay Whether or not steps should be placed above a darkened
    * modal overlay. If true, the overlay will create an opening around the target element so that it
    * can remain interactive
+   * @param {HTMLElement} options.modalContainer Optional container Element for the modal.
```

Comment:
Can you also document this here please?  https://github.com/shipshapecode/shepherd/blob/master/docs-src/tutorials/02-usage.md

Reply:
I will document it, no worries!

Reply:
thanks!